controllers/licenseissuer/internal/controller/launcher_controller.go

```diff
@@ -56,67 +57,96 @@ type LauncherReconciler struct {
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.14.4/pkg/reconcile
 func (r *LauncherReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctl ctrl.Result, err error) {
```

Comment:
this one too, add comments and split it.

Reply:
any updates?

Reply:
There is no good way to optimize this. The logic of this module is naturally very complex, and the read and write operations can't be effectively decoupled. Unless safety is sacrificed.